src/Blueprint/Blueprint/help/Az.Blueprint.md

```diff
@@ -1,28 +1,53 @@
 ---
 Module Name: Az.Blueprint
-Module Guid: {{ Update Module Guid }}
-Download Help Link: {{ Update Download Link }}
-Help Version: 1.0.0.0
-Locale: en-us
+Module Guid: ef36c942-4a71-4e19-9450-05a35843deb6
+Download Help Link: {{Please enter FwLink manually}}
+Help Version: {{Please enter version of help manually (X.X.X.X) format}}
+Locale: en-US
 ---
 
 # Az.Blueprint Module
 ## Description
 The topics in this section document the Azure PowerShell cmdlets for Azure Blueprints in the Azure Resource Manager framework. The cmdlets exist in the Microsoft.Azure.PowerShell.Cmdlets.Blueprint namespace.
 
 ## Az.Blueprint Cmdlets
+### [Export-AzBlueprintWithArtifact](Export-AzBlueprintWithArtifact.md)
+Export specified blueprint to the specified output location as a JSON file. 
+
 ### [Get-AzBlueprint](Get-AzBlueprint.md)
 Get one or more blueprint definitions.
 
+### [Get-AzBlueprintArtifact](Get-AzBlueprintArtifact.md)
+Get artifact from a blueprint.
+
 ### [Get-AzBlueprintAssignment](Get-AzBlueprintAssignment.md)
 Get one or more blueprint assignments.
 
+### [Import-AzBlueprintWithArtifact](Import-AzBlueprintWithArtifact.md)
+Import a blueprint file in JSON format to a blueprint object and save it within the specified subscription or management group.
```

Comment:
"Import a blueprint definition file..." and "... to a blueprint definition object..."